django/contrib/admin/filters.py

```diff
@@ -412,5 +412,12 @@ def choices(self, cl):
 
 class RelatedOnlyFieldListFilter(RelatedFieldListFilter):
     def field_choices(self, field, request, model_admin):
-        limit_choices_to = {'pk__in': set(model_admin.get_queryset(request).values_list(field.name, flat=True))}
-        return field.get_choices(include_blank=False, limit_choices_to=limit_choices_to)
+        limit_choices_to = {
+            'pk__in': set(
+                model_admin
+                .get_queryset(request)
+                .values_list('{0}__pk'.format(self.field_path), flat=True)
+            )
+        }
+        return field.get_choices(
```

Comment:
We use line lengths up to 119 characters when it helps readability. I wouldn't reformat this line when it doesn't have any other changes.